echo/echo_alias.go

```diff
@@ -0,0 +1,8 @@
+package echo
+
+import "github.com/labstack/echo/v4"
+
+// Context is an alias/composition of echo.Context that allows generating a mock of echo.Context
+type Context interface {
```

Comment:
Why do we need this? It wasn't required before?

Reply:
This mocks an external interface. This mock was probably created externally and then imported into our codebase, but this change allows regenerating it easily. The echo.Context interface has not changed, so the mock has not stopped working. 

However, this mock file can go as soon as all APIs are are converted to use strict OAPI interfaces.

Reply:
Add a comment to the type?

Reply:
rebased and removed changes to mock/ since it was starting to conflict with the strict openAPI interface PRs. Made #1827 to remove this later.